app/js/controllers/main.js

```diff
@@ -50,22 +48,34 @@ export default class MainController extends Controller {
       return;
     }
 
-    this.view.customizer.setRootNode(document.documentElement);
+    // Insert the customizer view into the app's document
     document.body.appendChild(this.view.el);
 
-    var padding = this.view.el.querySelector('fxos-customizer').offsetHeight;
-    this._styleEl = document.createElement('style');
-    this._styleEl.textContent = `
-    html, body {
-      overflow: initial !important;
-    }
+    // And display it. This has to be in a setTimeout or the CSS animation
+    // won't work correctly since we just added it above.
+    setTimeout(() => { this.view.show(); });
+
+    // Wait a bit before populating it and defining the new stylesheet
+    // so that we can just get it on the screen quickly and don't interfere
+    // with the animation
+    setTimeout(() => {
+      this.view.customizer.setRootNode(document.documentElement);
+
+      var padding = this.view.el.querySelector('fxos-customizer').offsetHeight;
+      this._styleEl = document.createElement('style');
```

Comment:
Its weird to have part of a view that affects the entire containing document. But I agree that it doesn't belong in the controller. I'll move it.  I'm considering just adding the stylesheet once and enabling and disabling it (I think we can do that) instead of recreating it each time.  We'd have to hardcode the padding height, but since the view will be hard-coding the position of the customizer, I don't see any problem also hard-coding the padding.